rakelib/plugin.rake

```diff
@@ -61,6 +61,8 @@ namespace "plugin" do
     Dir["logstash-core*.gem"].each do |gem|
       rm(gem)
     end
```

Comment:
waht do you mean all possible paths??? there is no double-star in the glob `**` ??